Задание для кандидатов. Выполнить ревью кода.sql

```diff
@@ -1 +1,118 @@
-
+create procedure syn.usp_ImportFileCustomerSeasonal
+	@ID_Record int

```

Comment:
```suggestion
	declare @ID_Record int

```
18. Перед переменно должно быть объявление declare.